README.md

```diff
@@ -20,6 +20,12 @@ Then choose from the following gulp tasks:
 * `gulp` - generates the build in the `./dist` folder and runs a development server on `localhost:3000`.
 * `gulp build` - generates the build in the `./dist` folder.
 
+### Run unit tests
+To run the unit tests execute the command:
+```
+npm test
```

Comment:
@pauli, can we run them via a gulp task? `gulp test`, that way we can also include it in our build tasks.

Reply:
This looks like it can help us out https://www.npmjs.com/package/gulp-jest